db/migrate/20210110100235_remove_unused_application_feedback_booleans_and_add_feedback_column.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveUnusedApplicationFeedbackBooleansAndAddFeedbackColumn < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :application_feedback, :does_not_understand_section, :boolean
+    remove_column :application_feedback, :need_more_information, :boolean
+    remove_column :application_feedback, :answer_does_not_fit_format, :boolean
+    add_column :application_feedback, :feedback, :string
```

Comment:
I think this is the missing line from the previous PR and needs to be moved back, so that it's deployed before you make reference to the new `feedback` column

Reply:
have sorted